solidity/contracts/isms/PausableIsm.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+// ============ External Imports ============
+import {Pausable} from "@openzeppelin/contracts/security/Pausable.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+// ============ Internal Imports ============
+import {IInterchainSecurityModule} from "../interfaces/IInterchainSecurityModule.sol";
+
+contract PausableIsm is IInterchainSecurityModule, Ownable, Pausable {
+    uint8 public constant override moduleType = uint8(Types.NULL);
+
+    /**
+     * @inheritdoc IInterchainSecurityModule
+     * @dev Returns `false` when paused, `true` otherwise.
+     */
+    function verify(bytes calldata, bytes calldata)
+        external
+        view
+        returns (bool)
+    {
+        return !paused();
```

Comment:
Wdyt about a revert message in here to indicate to users that it's because this ISM has been triggered?